src/Galley/PhysicalOptimizer/protocol-selector.jl

```diff
@@ -93,25 +93,25 @@ function modify_protocols!(expr)
                     index == var && break
                     push!(indices_before_var, index)
                 end
-                # The choice of `min` below is arbitrary because the actual agg_op doesn't affect
+                # The choice of `|` below is arbitrary because the actual agg_op doesn't affect
                 # the nnz (barring things like prod reductions which might be a TODO).
                 # TODO: Replace this with conditional estimates
                 if length(indices_before_var) > 0
                     size_before_var = estimate_nnz(
                         reduce_tensor_stats(
-                            min,
-                            typemax(get_default_value(input)),
+                            |,
+                            false,
                             setdiff(get_index_set(input), indices_before_var),
-                            input,
+                            set_fill_value!(input, false),
```

Comment:
@kylebd99 Here too, it seems the issue was not arbitrary ops, but arbitrary fill values that didn't get changed